pkg/tsdb/elasticsearch/response_parser_frontend_test.go

```diff
@@ -1391,24 +1391,24 @@ func TestRawData(t *testing.T) {
 	require.NoError(t, err)
 
 	require.Len(t, result.response.Responses, 1)
-	// frames := result.response.Responses["A"].Frames
-	// require.True(t, len(frames) > 0) // FIXME
+	frames := result.response.Responses["A"].Frames
+	require.True(t, len(frames) > 0)
 
-	// for _, field := range frames[0].Fields {
-	// 	trueValue := true
-	// 	filterableConfig := data.FieldConfig{Filterable: &trueValue}
+	for _, field := range frames[0].Fields {
+		trueValue := true
+		filterableConfig := data.FieldConfig{Filterable: &trueValue}
 
-	// 	// we need to test that the only changed setting is `filterable`
-	// 	require.Equal(t, filterableConfig, *field.Config) // FIXME
-	// }
+		// we need to test that the only changed setting is `filterable`
+		require.Equal(t, filterableConfig, *field.Config)
+	}
 }
 
-func TestLogsAndCount(t *testing.T) {
+func TestLogs(t *testing.T) {
```

Comment:
I am not sure why was this test structured this way on the frontend, but we basically in here https://github.com/grafana/grafana/blob/main/public/app/plugins/datasource/elasticsearch/ElasticResponse.test.ts#L1414 (the same test on frontend), we create a query that is not `logs` type, and then process it as logs. This seems like artificial test, because if the query is not `logs` query, then we shouldn't process it that way. And because all of the tests focus on `logs` logic, I have changed the query type to be `logs` and removed all `count` parts.

Reply:
Also on frontend, we have a messy logic, where we run `processResponseToDataFrames` and then use it for both logs and time series. So if `logs` test data have some aggregations, these are also processed. But this shouldn't be a case because `logs` query should be processed only as `logs` query